fanuc_lrmate200id_gazebo/launch/lrmate200id_control.launch

```diff
@@ -0,0 +1,17 @@
+<launch>
+
+  <!-- Load joint controller configurations from YAML file to parameter server -->
+  <rosparam file="$(find fanuc_lrmate200id_gazebo)/config/lrmate200id_controllers.yaml" command="load"/>
+
+  <!-- load the controllers -->
+  <node name="controller_spawner"
+        pkg="controller_manager" type="spawner"
+        args="joint_state_controller arm_controller"
+        respawn="false" output="screen" />
+
+  <!-- convert joint states to TF transforms for rviz, etc -->
+  <node name="robot_state_publisher"
+        pkg="robot_state_publisher" type="robot_state_publisher"
+        respawn="false" output="screen" />
```

Comment:
Can you explain why you chose to include the `robot_state_publisher` here? Starting that node is typically left as something that the caller is responsible for (as you don't know how he wants to configure it), and it makes this launch files less re-usable - it would be nice if we can use the `_control.launch` files as 'stand-in' driver launch files (ie: consider the gazebo robot a surrogate Fanuc).